units/host_termination.go

```diff
@@ -100,6 +100,13 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 		j.env = evergreen.GetEnvironment()
 	}
 
+	if j.host.IsStatic() {
```

Comment:
This is equivalent to `j.host.IsEphemeral`, so it can be absorbed into the block below this. It will also help future investigations to preserve the grip notice in case we one day somehow introduce a bug where static hosts get selected for automatic termination (rather than by manual action).